src/Pages/Experts/Component/Modal.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'styled-components';
```

Comment:
```suggestion
import React from 'react';
import styled from 'styled-components';
import { faTimes } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
import 순서 이전에 리뷰 드렸을 건데 혹시 순서 알고 계시나요?

Reply:
React
Library(Package)
Component
변수 / 이미지
css 파일(scss 파일)
이렇게라고 찾아보긴 했는데 아직 뭐가 뭔지 몰라서 정렬을 못한 거 같네요